src/buildah.ts

```diff
@@ -25,10 +26,28 @@ interface Buildah {
 export class BuildahCli implements Buildah {
     private readonly executable: string;
 
+    public storageOptsEnv = "";
+
     constructor(executable: string) {
         this.executable = executable;
     }
 
+    // Checks for storage driver if found "overlay",
+    // then checks if "fuse-overlayfs" is installed.
+    // If yes, add mount program to use "fuse-overlayfs"
+    async setStorageOptsEnv(): Promise<void> {
+        if (await isStorageDriverOverlay()) {
+            const fuseOverlayfsPath = await findFuseOverlayfsPath();
+            if (fuseOverlayfsPath) {
+                core.info(`Overriding storage mount_program with "fuse-overlayfs" in environment`);
+                this.storageOptsEnv = `overlay.mount_program=${fuseOverlayfsPath}`;
+            }
+            else {
+                core.warning(`"fuse-overlayfs" is not found. Install it before running this action`);
```

Comment:
This shouldn't be a warning; there are use-cases like our self-hosted runenr that use different drivers.

Reply:
`core.info("Storage driver is not 'overlay', so not overriding storage configuration")`